apps/api/src/app/events/services/sms-service/handlers/bulksms.handler.ts

```diff
@@ -0,0 +1,25 @@
+import { ChannelTypeEnum } from '@novu/shared';
+import { ICredentials } from '@novu/dal';
+import { BulkSmsSmsProvider } from 'D:/Novu_latest/novu/providers/bulkSms/src/lib/bulkSms.provider';
+import { BaseSmsHandler } from './base.handler';
+
+export class BulkSmsSmsHandler extends BaseSmsHandler {
+  constructor() {
+    super('bulksms', ChannelTypeEnum.SMS);
+  }
+  buildProvider(credentials: ICredentials) {
+    const config: {
+      authKey?: string;
+      sender?: string;
+      route?: string;
+      // dltEntityId?: string;
+    } = {
+      authKey: credentials.authKey,
+      sender: credentials.sender,
```

Comment:
Can we reuse the from credentials.from here?
That way we won't need an additional variable with the same meaning on the ICredentials interface. WDYT? 

```suggestion
      sender: credentials.from,
```

Reply:
Yes We can do that

Reply:
This hasn't been addressed yet.

Reply:
> sender: credentials.from

@p-fernandez have used sender: credentials.from and I have also pushed the same. Hope it will help you out.